### PR TITLE
BITMAKER-2060 Set Up Estela For Hacktoberfest Contributions

### PR DESCRIPTION
**Pull request by raymond1242**
# Description

Set Up Estela For Hacktoberfest Contributions

# Issue

https://tasks.bitmaker.dev/issues/2060

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] My code follows the style guidelines of this project.
- [x] I have made corresponding changes to the [documentation](https://github.com/bitmakerla/estela/tree/main/docs).
- [x] New and existing tests pass locally with my changes.
- [x] If this change is a core feature, I have added thorough tests.
- [x] If this change affects or depends on the behavior of other _estela_ repositories, I have created pull requests with the relevant changes in the affected repositories. Please, refer to our [official documentation](https://estela.bitmaker.la/docs/).
- [x] I understand that my pull request may be closed if it becomes obvious or I did not perform all of the steps above.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Issue that pull request!
+
+## Any contributions you make will be under the MIT Software License
+In short, when you submit code changes, your submissions are understood to be under the same [MIT License](http://choosealicense.com/licenses/mit/) that covers the project. Feel free to contact the maintainers if that's a concern.
+
+## Report bugs using Github's [issues](https://github.com/bitmakerla/estela/issues)
+We use GitHub issues to track public bugs. Report a bug by [opening a new issue](); it's that easy!
+
```
Comment (Adriana618) [inline CONTRIBUTING.md:25]: Link doesn't work

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+We use [black](https://github.com/psf/black) code formatter
+
+* Run `black .` for style unification
+
+## License
+By contributing, you agree that your contributions will be licensed under its MIT License.
+
+## References
+This document was adapted from the open-source contribution guidelines for [Facebook's Draft](https://github.com/facebook/draft-js/blob/a9316a723f9e918afde44dea68b5f9f39b7d9b00/CONTRIBUTING.md)
```
Comment (Adriana618) [inline CONTRIBUTING.md:48]: Link doesn't work.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Use [Github Flow](https://github.com/bitmakerla/estela/actions), So All Code Changes Happen Through Pull Requests
+Pull requests are the best way to propose changes to the codebase. We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
+2. If you've added code that should be tested, add tests.
```
Comment (Adriana618) [inline CONTRIBUTING.md:14]: do We have a structure for tests? 
In what file or folder the contributors add the tests?
What is the approach to make test?
This and other questions needs to be answered.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
```
Comment (Adriana618) [inline CONTRIBUTING.md:16]: How they update the documentation?
We need to define steps for this.

**Review by Adriana618 — CHANGES_REQUESTED** [review]
Maybe you can check this [guide](https://mozillascience.github.io/working-open-workshop/contributing/) about to create a CONTRIBUTING.md, for example give _rewards_ for contribute or fork is a good way to encourage people to contribute.

**Comment by raymond1242** [comment]
@Adriana618 Could you check this PR pls

**Review by essiocolom — APPROVED** [review]

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+## How to make Pull Requests
+
+We implemente a [Pull Request template](https://github.com/bitmakerla/estela/blob/main/.github/pull_request_template.md) that will be used in every request proposed. Just complete the template. 
+
+### Don't forget to reference the issue that you are working on.
+
+### Documentation updated
+To keep **estela** documented make sure you comment every funcionality added if it's necessary. Write the comment above that funcionality
+
```
Comment (Adriana618) [inline CONTRIBUTING.md:28]: I don't know if this is the best way to keep code documented.

**Review by Adriana618 — APPROVED** [review]

**Pull request merged**